### ISSUE TITLE
Internal console complains about psutil module not having 'phymem_usage' attribute

### ISSUE THREAD

**Issue by narunask** [spyder-ide/spyder#3191]
## Description of your problem

**What steps will reproduce the problem?**

Start up `Spyder` and watch `Internal console`

**What is the expected output? What do you see instead?**

Clean console.

**Please provide any additional information below**

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/spyderlib/widgets/status.py", line 89, in update_label
    self.label.setText('%d %%' % self.get_value())
  File "/usr/lib/python2.7/dist-packages/spyderlib/widgets/status.py", line 102, in get_value
    return memory_usage()
  File "/usr/lib/python2.7/dist-packages/spyderlib/utils/system.py", line 44, in psutil_phymem_usage
    return psutil.phymem_usage().percent
AttributeError: 'module' object has no attribute 'phymem_usage'
```

## Versions and main components

* Spyder Version: spyder 2.3.1
* Python Version: Python 2.7.11+
* Operating system:
```
Distributor ID: Debian
Description:    Debian GNU/Linux testing (stretch)
Release:        testing
Codename:       stretch
```
* psutils 1.17
* python-psutil 4.1.0-1
* python3-psutil 4.1.0-1

## Dependencies

IPython >=0.13  :  2.4.1 (OK)
matplotlib >=1.0:  1.5.1 (OK)
pandas >=0.13.1 :  0.18.0+git114-g6c692ae (OK)
pep8 >=0.6      :  None (NOK)
psutil >=0.3    :  4.1.0 (OK)
pyflakes >=0.5.0:  1.2.3 (OK)
pygments >=1.6  :  2.1.3 (OK)
pylint >=0.25   :  1.5.5 (OK)
rope >=0.9.2    :  0.10.2 (OK)
sphinx >=0.6.6  :  1.3.6 (OK)
sympy >=0.7.0   :  None (NOK)

**Comment by ccordoba12** [comment]
Thanks for reporting. This problem seems to be related to an API change in psutil.

**Comment by narunask** [comment]
Thanks for reviewing the bug. I understand status of this bug for version 2.x - `won't fix`? If so, can you suggest any workaround? It does not affect much of my workflow, but notwithstanding, it's quite annoying, as I can't use console from within the `spyder`.

**Comment by Nodd** [comment]
You can deactivate the CPU load tracking in the preferences.
Why can't you use the console ? The message appears in the internal console that you shouldn't use anyway.

**Comment by narunask** [comment]
`Internal console` tab goes to the foreground when error trace is printed, essentially `python console` tab then is put to background.

**Comment by Nodd** [comment]
Right, I forgot about that. There is an option in the preferences to ignore internal errors. Both should be in the first tab (sorry for not being more precise, spyder is broken on Qt5.6 on my computer).

**Comment by ccordoba12** [comment]
This problem was fixed in Spyder 2.3.2. This is the reference: ae5e00511ca497d103d67c1d5ea00d4cc59f1434

Please update your Spyder version.
